### DISCUSSION TITLE
Is there a way to modify

### DISCUSSION THREAD

**Discussion by brennanfife** [family/connectkit#67] [Q&A] [answered]
This may be a duplicate of issue #62, but is there an easy way to modify the values shown on light vs. dark mode?

**Comment by brennanfife** [comment]
@lochie

**Comment by META-DREAMER** [comment]
I am setting all the values to other CSS variables that I've created that change value based on light or dark mode (using Radix colors)

![CleanShot 2022-12-14 at 16 24 16](https://user-images.githubusercontent.com/7143583/207736789-c216da6c-2a79-4972-a2e7-97a07bc090a8.png)

**Comment by lochie** [comment, marked as answer]
Custom Theme Variables are still an experimental feature of ConnectKit. We currently don't support theme mode switching with Custom Themes Variables but this is definitely something we'll be looking into right away when we start extending out this API.

@META-DREAMER's workaround is the best solution for the time being (thanks for that!)

I've also started [a discussion](https://github.com/family/connectkit/discussions/77) as a feature request.
